### PR TITLE
ignore proto expectation errors from traceviewer

### PR DESCRIPTION
**Pull request by paulirish**
(really, all of the console.error() calls are muted by this)

I am replacing with `console.assert(true, ...)` just to not overcomplicate the replacement.

### CODE REVIEW

**Comment by wardpeet** [comment]
Looks ok to me, not much has changed

**Comment by patrickhulce** [comment]
works for me

File: lighthouse-core/scripts/build-traceviewer-module.js [hunk 1]
```diff
@@ -70,8 +70,9 @@ function convertImport(src) {
         scriptsContent += rewriteGlobals(scripts[s]);
       }
 
+      scriptsContent = silenceTheConsole(scriptsContent);
       // node4 compat
-      scriptsContent = polyfillNode4Support(dest, scriptsContent);
+      scriptsContent = polyfillNode4SupportAndMinify(dest, scriptsContent);
```
Comment (brendankenny) [inline lighthouse-core/scripts/build-traceviewer-module.js]: can we remove this now with no node 4?

Comment (paulirish) [inline lighthouse-core/scripts/build-traceviewer-module.js]: yup. will do with a quick followup.

**Pull request merged**